openedx_caliper_tracking/pavelib/settings.py

```diff
@@ -0,0 +1,8 @@
+"""Settings related to or required by all paver tasks"""
+import os
+
+CALIPER_TESTS_API = 'CALIPER_TESTS_API'
+
+
+CALIPER_TESTS_API_ADDRESS = os.environ.get(CALIPER_TESTS_API)
+CALIPER_TESTS_API_TOKEN = os.environ.get('{}_TOKEN'.format(CALIPER_TESTS_API))
```

Comment:
Don't format this string. It causes unnecessary confusion.

Reply:
updated